src/Entity/Project/Ce.php

```diff
@@ -13,13 +13,19 @@
 
 use App\Entity\Personne\Personne;
 use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
 
 /**
+ * BdC and Ce (still named Ce for backward compatibility)
```

Comment:
Could be more detailled.

named CE because for implementation reasons (almost same object, makes less duplicated code), not really for any backward compatibility

Reply:
C’est vrai. J’ai eu une période où j’argumentais beaucoup avec des "backward compatibility" sans trop réfléchir… Oups